backend/src/mqttsServer.ts

```diff
@@ -16,7 +16,7 @@ const mqttsServer = (): Promise<Aedes> => {
     broker.on('clientError', (client: Client, error: Error) => {
       console.log(
         `MQTT client \x1b[34m${client ? client.id : client}\x1b[0m  receive this error   \x1b[34m${JSON.stringify(
-          error,
+          error.message,
```

Comment:
Instead of only displaying the `message` property and losing the `name` and `stack` properties. Is a good idea to use [inspect()](https://nodejs.org/api/util.html#utilinspectobject-options), like here: https://github.com/m8schmit/ecovacs-stack/blob/main/backend/src/mqttClient/mqtt.utils.ts#L29 ?

Reply:
That looks even better, yes indeed!
I updated the PR to use it.

PS: As I'm new to typescript, I was not sure how to best indent it, sorry.

Reply:
Their an [eslint](https://eslint.org/) and a [prettier](https://prettier.io/) plugin installed in the project to think to this for you. You need to configure your IDE to use them.